packages/grid/_modules_/grid/locales/daDK.ts

```diff
@@ -0,0 +1,127 @@
+import { daDK as daDKCore } from '@mui/material/locale';
+import { GridLocaleText } from '../models/api/gridLocaleTextApi';
+import { getGridLocalization, Localization } from '../utils/getGridLocalization';
+
+const daDKGrid: Partial<GridLocaleText> = {
+  // Root
+  noRowsLabel: 'Ingen rækker',
+  noResultsOverlayLabel: 'Ingen resultater',
+  errorOverlayDefaultLabel: 'Der skete en fejl.',
+
+  // Density selector toolbar button text
+  toolbarDensity: 'Density',
```

Comment:
According to Google Translate, this is not Danish
Same for others below.
Could @kasperfilstrup indicate the untranslated one and either translate them or comment them ?

Reply:
Good catch. I commented on lines that are in English

Reply:
@kasperfilstrup if you want to modify translation to this PR, the easiest way is to make suggestions on the [change files](https://github.com/mui-org/material-ui-x/pull/3800/files)

![Screenshot from 2022-01-31 14-09-17](https://user-images.githubusercontent.com/45398769/151799395-5e3228bf-5c4c-476e-808d-2110cd945314.png)